### ISSUE TITLE
SVD segfaults on 2x2 matrices

### ISSUE THREAD

**Issue by bbrener1** [rust-ndarray/ndarray-linalg#151]
https://github.com/bbrener1/dylib_linking_test

Minimum reproducible case in repo above

When SVD is performed on a 2x2 matrix, for example a an identity matrix containing f64s, the program terminates with a segfault. 

I have reproduced this bug on Ubuntu and OSX, and have reproduced it using Intel-MKL and OpenBlas backends. 

Bug is also apparent in a docker container using run -it rust:latest, so versioning is unlikely to be at issue. 

Please see if you can reproduce and track down the cause

**Comment by jturner314** [comment]
Thanks for reporting this and for providing an easy-to-reproduce case!

It looks like this was fixed in #130. To verify this, you can add the following to `Cargo.toml`

```rust
[patch.crates-io]
ndarray-linalg = { git = "https://github.com/jturner314/ndarray-linalg.git", branch = "revert-version-number" }
```

and re-run the example.

This fix will be part of the next release of `ndarray-linalg`.

**Comment by bbrener1** [comment]
Patch fixes issue, thank you for your help! Looking forward to next release.
